src/app/modules/project/add/manual/manual.component.html

```diff
@@ -84,6 +83,30 @@ <h3>Project icon</h3>
       </div>
     </form>
 
+    <form [formGroup]="newCallToActionForm" class="project-form">
+
+      <div class="row">
+        <div class="col-12">
+          <h3>Call to action</h3>
+        </div>
+        <div class="col-6">
+          <div>
+            <h3>Text</h3>
+            <select [(ngModel)]="selectedCallToActionOption" class='form-control' [ngModelOptions]="{standalone: true}">
+              <option *ngFor="let action of callToActionOptions" [ngValue]="action">{{action.value}}</option>
+            </select>
+          </div>
+        </div>
+
+        <div class="col-6">
+          <div>
+            <h3>Link</h3>
+            <input [(ngModel)]="callToActionRedirectUrl" [ngModelOptions]="{standalone: true}" [disabled]="selectedCallToActionOption.id <= 0" placeholder="Link to your contact info"/>
+          </div>
+        </div>
+      </div>
+    </form>
```

Comment:
Can we perhaps add a helper description that explains what 'call to actions' are for? It might make it easier for users to understand what they are here for and what you can do with it.

Reply:
Please note that this feedback is applicable to editing a project as well.